crates/biome_cli/tests/snapshots/main_commands_rage/with_linter_configuration.snap

```diff
@@ -76,7 +76,7 @@ Linter:
                                 correctness/all = true
                                 nursery/useConsistentArrayType = {"level":"warn","options":{"syntax":"shorthand"}}
                                 style/noNonNullAssertion = "off"
-                                suspicious/noCommentText = {"level":"warn"}
+                                suspicious/noCommentText = {"level":"warn","options":null}
```

Comment:
I didn't manage to skip serialization if the type is `()`. This is not so important since it is unlikely that a user use `RuleWithOptions` when a rule doesn't accept any options.

Reply:
I guess a trick like `TypeId::of::<T>() == TypeId::of::<()>()` might work?